graphql-database-dynmodb-history-lambda/src/main/java/com/fleetpin/graphql/database/dynamo/history/lambda/DynamoUtil.java

```diff
@@ -0,0 +1,97 @@
+package com.fleetpin.graphql.database.dynamo.history.lambda;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent.DynamodbStreamRecord;
+
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.Identity;
+import software.amazon.awssdk.services.dynamodb.model.Record;
+import software.amazon.awssdk.services.dynamodb.model.StreamRecord;
+
+public class DynamoUtil {
+
+	public static Record toV2(DynamodbStreamRecord record) {
+
+		var builder = Record.builder();
+		builder.awsRegion(record.getAwsRegion());
+		builder.dynamodb(toV2(record.getDynamodb()));
+		builder.eventID(record.getEventID());
+		builder.eventName(record.getEventName());
+		builder.eventSource(record.getEventSource());
+		builder.eventVersion(record.getEventVersion());
+		builder.userIdentity(toV2(record.getUserIdentity()));
+		return builder.build();
+	}
+
+	public static Identity toV2(com.amazonaws.services.dynamodbv2.model.Identity userIdentity) {

```

Comment:
Don't need to use the fully qualified class name.

Reply:
one on the left is software.amazon different classes